src/Selenium2Driver.php

```diff
@@ -423,7 +423,17 @@ public function switchToWindow($name = null)
      */
     public function switchToIFrame($name = null)
     {
-        $this->wdSession->frame(array('id' => $name));
+        if (!empty($name)) {
+          $element = $this->wdSession->element('name', $name);
+          $element_info = [
+            'ELEMENT' => $element->getID(),
```

Comment:
Thank you for this PR, it will solve https://github.com/minkphp/driver-testsuite/pull/60

`'ELEMENT'` key is not compatible with W3C spec, see https://github.com/instaclick/php-webdriver/commit/b5b18bd2e02b35a14c7085552780588b8876460b

Will tests pass if you pass `$element` directly like `$this->wdSession->frame(array('id' => $element))`?

Reply:
> Will tests pass if you pass `$element` directly like `$this->wdSession->frame(array('id' => $element))?`

No, tests will fail, because `instaclick/php-webdriver` doesn't support this. The https://github.com/instaclick/php-webdriver/pull/132 should add this functionality.

In the meanwhile the workaround was created in the https://github.com/minkphp/MinkSelenium2Driver/pull/382.